packages/fiori/src/Bar.js

```diff
@@ -133,6 +143,49 @@ class Bar extends UI5Element {
 			"label": this.design,
 		};
 	}
+
+	constructor() {
+		super();
+
+		this._handleResizeBound = this.resizeHandler.bind(this);
+	}
+
+	resizeHandler() {
```

Comment:
better function names would be: handleResize, onResize

Reply:
Done